docs/modules/ROOT/pages/headers.adoc

```diff
@@ -0,0 +1,54 @@
+# DataSonnet Header
+
+The header is an optional comment section in the mapping document which must start with `/\** DataSonnet` followed by a new line character and end with a line `*/` and contains a set of configuration properties.
+The properties are hierarchical, with sections separated by the `.` (dot) character, e.g.:
+
+------------
+dataformat.application/xml.payload.NamespaceSeparator=:
+dataformat.application/xml.payload.TextValueKey=__text
+dataformat.application/csv.myVar.Quote="
+dataformat.application/csv.myVar.Separator=|
+------------
+
+The header must be the first comment before any other line of code, in the following format:
+
+-------------
+/** DataSonnet
+version=<VERSION_NUMBER>
+<PROPERTY>=<VALUE>
+...
+*/
+-------------
+*Example*
+------------------------
+/** DataSonnet
+version=1.0
+*/
+//DataSonnet mapping starts here
+{
+    mappedValue: payload.value
+}
+------------------------
+
+The header is optional; if omitted, all default settings will be used.
```

Comment:
We should provide what the exact "default header" is

Reply:
I'd still like this (basically, just the version, but also a place if we add other parts of the header that are defaults)

Reply:
OK, so the default settings are defined in the plugins, not much in the header itself. OK, I can mention that the default version is 1.0 and other default settings are per plugin.